_content/reference/tigress-on-the-shwedagon.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "A Tigress on the Shwedagon: A Research Note"
+external_url: "https://www.burmalibrary.org/docs11/SBBR6-Tigress.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1xlDJ_i9_GO9k_xHOdhYUv41rlB2lOnFV/view?usp=sharing"
+course: burmese
+tags:
+  - ethics
+year: 2008
+journal: "SOAS Bulletin of Burma Research"
```

Comment:
If it has a journal, then articles for sure 😊  Even if it doesn't have a number / volume (year is enough)